src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -0,0 +1,21 @@
+import {addMonths, format, isBefore} from 'date-fns';
+import CONST from '@src/CONST';
+
+/**
+ * Get the next billing date.
+ *
+ * @param initialDate - The initial billing date in 'yyyy-MM-dd' format.
+ * @returns - The next billing date in 'yyyy-MM-dd' format.
+ */
+function getNextBillingDate(initialDate: string): string {
+    const start = new Date(initialDate);
```

Comment:
Should we add some validations for dates here? My experience with `date-fns` is that if the date is invalid, it'll throw an error.

Reply:
Yes to validation as a best practice if it's not too challenging, although the data from the BE should be pretty uniform so it'd have to be something very weird happening.

Also I mentioned this in [another PR](https://github.com/Expensify/App/pull/42990#discussion_r1631322823) and @MrMuzyk said he might create a util, but due to timezone differences I think at least for now we should pass `T00:00:00` so that the date is properly calculated (e.g. for me if my start date is 6/1/24 the next bill date should be 7/1/24, but it's currently showing 6/30/24)

Reply:
Updated with validation and proper billing day